client/lib/abtest/active-tests.js

```diff
@@ -73,4 +73,13 @@ module.exports = {
 		defaultVariation: 'showMonthly',
 		allowExistingUsers: true
 	},
+
+	signupCopy: {
+		datestamp: '20170307',
+		variations: {
+            oldSignupCopy: 50,
```

Comment:
I think a simple `new`/`old` (or `original`/`modified`) should be enough - but it's up to you.